src/v/kafka/server/group.h

```diff
@@ -512,8 +519,12 @@ class group {
     config::configuration& _conf;
     ss::lw_shared_ptr<cluster::partition> _partition;
     absl::node_hash_map<model::topic_partition, offset_metadata> _offsets;
+    model::violation_recovery_policy _recovery_policy;
 
+    mutex _tx_mutex;
     model::term_id _term;
+    absl::flat_hash_map<model::producer_id, model::producer_epoch>
```

Comment:
we found node_hash behaves better w/ large sets and has minimal perf diff for our workloads.